ui/src/message_popup/renderers/question_interpreter.jsx

```diff
@@ -57,6 +59,28 @@ export default React.createClass({
     }
 
 
+    if (question.writeLong) {
```

Comment:
For this, I'd recommend changing `writeLong` here and in the scenario file to `notes`.  That captures the meaning of what should be happening, and then the details of how that gets shown happen to involve that it has a bigger text box.